tests/kafkatest/services/performance/producer_performance.py

```diff
@@ -62,6 +63,7 @@ def parse_stats(line):
             }
         last = None
         for line in node.account.ssh_capture(cmd):
+            self.maybe_start_jmx_tool(idx, node)
```

Comment:
Yes I have thought of the approach as you described -- I didn't implement it because it required the test case developer to specifically redirect output to a file and read from that file, which should not be required for developer to do if we can the right support from Ducktape. This is the reason I asked for asynchronous ssh in Ducktape during our discussion in the email. 

I am thinking of something like this:

```
cmd = "same_as_before"
output = node.account.ssh_asynchrous(cmd)
self.start_jmx_tool(node)
for line in output:
   # same as the previous for loop
```
Does it make sense?